tests/unit/misc/test_miscwidgets.py

```diff
@@ -137,8 +137,9 @@ def test_text(self, qtbot, config_stub, key_config_stub, bindings, text):
         qtbot.add_widget(w)
         assert w.text() == text
 
-    def test_timeout(self, qtbot, key_config_stub):
+    def test_timeout(self, qtbot, config_stub, key_config_stub):
+        config_stub.val.content.fullscreen.overlay_timeout = 1235
         w = miscwidgets.FullscreenNotification()
         qtbot.add_widget(w)
-        with qtbot.waitSignal(w.destroyed):
-            w.set_timeout(1)
+        with qtbot.waitSignal(w.destroyed, timeout = 1236):
```

Comment:
I'm surprised that this works on Travis CI right now - if the signal is delayed by only 1ms (or the time between signal and destruction takes 1ms), this will fail. Also, it means the test will just spend >1s waiting, which makes the test a lot slower. I'd suggest instead setting the timeout to `1` again, because we can't really test the setting, and this way, the test will finish as fast as possible.

I think that means you can just revert the changes for the test entirely - we're only testing `miscwidgets.FullscreenNotification()` here, which doesn't read the setting itself.